lms/views/admin/__init__.py

```diff
@@ -1,10 +1,34 @@
+from marshmallow import fields, missing
 from pyramid.httpexceptions import HTTPFound, HTTPNotFound
 from pyramid.renderers import render_to_response
 from pyramid.view import forbidden_view_config, notfound_view_config, view_config
 
 from lms.validation._exceptions import ValidationError
 
 
+class EmptyStringNoneMixin:
```

Comment:
This is very verbose but I reckon we should handle any of this validations with Marshmallow.